website_legal_page/tests/test_controller.py

```diff
@@ -0,0 +1,30 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017 LasLabs Inc.
+# License APL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo.tests.common import HttpCase
+from odoo.tools import mute_logger
+
+
+class TestController(HttpCase):
+    def _test_page(self, page, code=200):
+        response = self.url_open(page, timeout=20)
+        self.assertEqual(response.status_code, code)
+
+    # TODO: comment this test because odoo 11 return 200 even if page does not exist..
```

Comment:
Whaaat? Seriously? :smile:

Reply:
Thanks :)

Reply:
Instead of this, you should:

```python
# HACK https://github.com/odoo/odoo/issues/22438
# TODO Remove when fixed
@unittest.expectedFailure
#[and here the test, uncommented]

Reply:
@njeudy it's fixed. Would you mind dropping the hack? :)